CLAUDE.md

```
@@ -5,31 +5,42 @@ This file provides guidance to Claude Code (claude.ai/code) when working with co
+genlayer network                               # Select network
+genlayer deploy                                # Deploy contracts
+
+# Frontend
+cd frontend && npm run dev                     # Start frontend dev server
 ```
 
 ## Architecture
 
 ```
 contracts/          # Python intelligent contracts
+tests/
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)